btree/btree.c

```diff
@@ -9813,8 +9813,9 @@ static struct m0_atomic64 threads_running;
 		bool try_again;                                               \
 		do {                                                          \
 			try_again = false;                                    \
-			if (m0_atomic64_cas(&thread_run, true, false)) {      \
-				while (m0_atomic64_get(&threads_quiesced) <    \
+			if (m0_atomic64_cas((int64_t *)&thread_run, true,     \
```

Comment:
Please assign/check `0,1` instead of `true/false` for int64_t variable.

Reply:
Done.